app/resources/locations.json

```diff
@@ -0,0 +1,2047 @@
+[
```

Comment:
See my comment about not reinventing the wheel

Reply:
i use python module to find thee location by ip.but i need to create class location to my locattion file-locations that fit to api that i used with.the file is to compare to the ip location